docs/specs/meta-clients/web3wallet/sdk-api.md

```diff
@@ -71,7 +71,7 @@ class Web3Wallet {
   public abstract getPendingAuthRequests(): Promise<Record<number, PendingRequest>>;
   
   // register device token for Push server (BOTH)
-  public abstract registerDeviceToken(token: string): Promise<void>;
+  public abstract registerDeviceToken(token: string, enableAlwaysDecrypted: boolean?): Promise<void>;
```

Comment:
since this is an addition, it doesn't need to be nullable. The flag will be missing only on older clients

Reply:
@chris13524 do you have any objections to making this non-nullable on the server side?

Reply:
`always_null` in the push server API needs to be nullable because it needs to support clients who have not updated the SDK. `null` is interpreted as `false`

Reply:
I agree with @arein, `enableAlwaysDecrypted` should default to `true` if not specified

Reply:
But there's a problem with defaulting `enableAlwaysDecrypted` to `true` because it means that we cut off the legacy users when they upgrade their SDK.
The upgrade path for wallets is to set this flag to true **AND** set up a background service
If they only set a flag without a background service, they would stop receiving legacy notifications